stylesheet.css

```diff
@@ -0,0 +1,101 @@
+*{
+    box-sizing:border-box;
+}
+
+body {
+    background-color :#f2f2f2;
+    font-size: 62.5%;
+    font-family: HKGrotesk, sans-serif;
+    height: 100vh;
+    width: 100vw;
+    position: relative;
+}
+.hometop {
+    background-color: #F8D5E2;
+    height: 50vh;
+    width: 100vw;
+    position:relative;
+    border-top: 5px solid #f2f2f2;
+    border-left: 5px solid #f2f2f2;
+    border-right: 5px solid #f2f2f2;
+
+}
+nav {
+    width: 98%;
```

Comment:
spoke about the maybe 100% width

Reply:
Rethought it, parent (Hometop) width is set to 100vw which is 100% basically . Left nav 98% as it doesn't affect the page width.